mypy/main.py

```diff
@@ -112,8 +112,11 @@ def process_options() -> Tuple[List[BuildSource], Options]:
             parsed flags)
     """
 
+    # Make the help output a little less jarring.
+    help_factory = (lambda *a, **k:
+                    argparse.RawDescriptionHelpFormatter(max_help_position=28, *a, **k))
```

Comment:
I assume it's because the `ArgumentParser` constructor also passes in a `max_help_position`.